api/db/postgres/postgres_handler.go

```diff
@@ -0,0 +1,48 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"sync"
+	"time"
+	_ "github.com/lib/pq" // Defining postgres plugin
```

Comment:
Would you mind removing this comment, please? 🙃

Reply:
It is necessary for the lint. You need to justify the if you import a package with `_` reference :(

The following message is generated if I remove the comment:
```
exported method PostgresRequests.ConnectDB should have comment or be unexported
```
I don't have option not to import the psql plugin. However, I don't it explictly. So I need to solve this using the comment.